### ISSUE TITLE
SCIP installation error

### ISSUE THREAD

**Issue by ChrisNabold** [SCIP-Interfaces/SCIP.jl#162]
I try to install SCIP, but I receive the following error
              _
   _       _ _(_)_     |  Documentation: https://docs.julialang.org
  (_)     | (_) (_)    |
   _ _   _| |_  __ _   |  Type "?" for help, "]?" for Pkg help.
  | | | | | | |/ _` |  |
  | | |_| | | | (_| |  |  Version 1.4.1 (2020-04-14)
 _/ |\__'_|_|_|\__'_|  |  Official https://julialang.org/ release
|__/                   |

(@v1.4) pkg> add SCIP
   Updating registry at `C:\Users\Chris\.julia\registries\General`
   Updating git-repo `https://github.com/JuliaRegistries/General.git`
  Resolving package versions...
  Installed SCIP - v0.7.4
   Updating `C:\Users\Chris\.julia\environments\v1.4\Project.toml`
  [82193955] + SCIP v0.7.4
   Updating `C:\Users\Chris\.julia\environments\v1.4\Manifest.toml`
  [82193955] + SCIP v0.7.4
   Building SCIP ? `C:\Users\Chris\.julia\packages\SCIP\lczBF\deps\build.log`
+ Error: Error building `SCIP`:
¦ ERROR: LoadError: SCIP is currently not supported on "NT"
¦ Stacktrace:
¦  [1] error(::String) at .\error.jl:33
¦  [2] top-level scope at C:\Users\Chris\.julia\packages\SCIP\lczBF\deps\build.jl:19
¦  [3] include(::String) at .\client.jl:439
¦  [4] top-level scope at none:5
¦ in expression starting at C:\Users\Chris\.julia\packages\SCIP\lczBF\deps\build.jl:14
+ @ Pkg.Operations C:\Julia-1.4.1\share\julia\stdlib\v1.4\Pkg\src\Operations.jl:892

**Comment by rschwarz** [comment]
Duplicate of #154 

As written there, you have SCIP.jl v0.7.4 installed, which does not support Windows. You should upgrade SCIP.jl. There might be some other packages installed with conflicting version requirements.

**Comment by ChrisNabold** [comment]
In order to insctall SCIP.jl I added SCIP.jl in Julia 1.4. Whatever I try Julia downloads Julia.jl V0.7.4. I removed SCIP.jl and reinstalled the interface it still loads V0.7.4. 

The variable SCIPOPTDIR is set to c:\Program files\SCIPOptSuite 7.0.0

Whatever I try scip downloads version 0.7.4. How can I upgrade to V0.7.5 to tra the interface to V7.0.0

 

Thank you for your help

Best regards 

Chris Nabold
